_sources/scripts/functions.sh

```diff
@@ -27,3 +28,36 @@ make_dir_writable() {
          ')' \
          -exec chgrp "$WWW_GROUP" {} \; -exec chmod g=rwX {} \;
 }
+
+export_vars_from_docker_secret_files() {
+    [ -f /etc/environment_secrets ] && rm /etc/environment_secrets
```

Comment:
what is this line about `/etc/environment_secrets` meant to be doing, and why? Please document, it isn't obvious from context